VoiceMemos/Controller/VoicesTableViewController.swift

```diff
@@ -310,10 +293,35 @@ class VoicesTableViewController: BaseTableViewController, UISearchBarDelegate {
         }
     }
     
-    override func tableView(tableView: UITableView, editingStyleForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCellEditingStyle {
-        return .Delete
+    override func tableView(tableView: UITableView, editActionsForRowAtIndexPath indexPath: NSIndexPath) -> [UITableViewRowAction]? {
+        let voice = self.voiceForRowAtIndexPath(indexPath, WithTableView: tableView)
+        
+        let deleteAction = UITableViewRowAction(style: .Normal, title: "Delete"){ action, index in
```

Comment:
`let deleteAction = UITableViewRowAction(style: .Normal, title: "Delete")SPACE_HERE{ action, index in`